nautobot/extras/tests/test_filters.py

```diff
@@ -652,6 +653,9 @@ class GraphQLTestCase(FilterTestCases.NameOnlyFilterTestCase):
     queryset = GraphQLQuery.objects.all()
     filterset = GraphQLQueryFilterSet
 
+    # skip testing query attribute for q filter test
+    q_filter = {"name": "icontains"}
```

Comment:
I don't understand from the comment alone why this is needed here - can you perhaps elaborate?

Reply:
The other field in the q filter is the `query` and the dynamic updates to the fields are not compatible with the graphql query format

Reply:
```suggestion
    # skip testing "query" attribute for generic q filter test as it's not trivially modifiable
    q_filter = {"name": "icontains"}
```